src/content/docs/style-guide/writing-docs/processes-procedures/create-preview-document.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Create content for the preview docs site
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+redirects:
+---
+
+The preview docs site hosts preview docs for upcoming features that we share with customers.
+
+Unlike the public docs site, the preview docs site is not publicly editable and does not appear in search engine results. Additionally, the site is not indexed, so you can't navigate from one preview doc to another. Because the preview docs site is located in a different repo than the public docs site, contributing looks a little different.
+
+Before you get started, make sure you have done the following:
+
+## Before you write [#do-first]
+
+- Verify that you have access to the [beta-docs-site repo](https://github.com/newrelic/beta-docs-site). If you do not  have access, contact the #doc-eng-help Slack channel for help.
+- Clone the beta-docs-site repo on your computer.
+- Make sure you're working in the correct repo on Github Desktop.
+- Create a branch off the beta-docs-site repo to store your work.
+  
+## Create a new preview doc [#create-docs]
+
+Once in the beta-docs-site repo:
+
+1. In `/src/content/docs/`, find a good location for your doc.
+2. Using your text editor, create a new `.mdx` file or copy an existing doc.
+3. Write your content, making sure to include any info about accessing the preview feature you're documenting.
+4. Commit your changes and create a pull request.
+5. Share the preview link with reviewers and implement any feedback.
+6. Merge the final PR.
+7. Create the official link by pasting the path name after https://docs-preview.newrelic.com/docs/.(Example: https://docs-preview.newrelic.com/docs/new-relic-distribution-of-opentelemetry)
+8. Provide reviewers the official link to share with customers.
+
+## Optional boiler plate text [#boiler-plate]
+
+When writing a doc for preview, you may want to include a [callout](/docs/style-guide/formatting/callouts/) with boiler plate text at the top of the doc. This text orients your reader, providing them with important information about the feature's availability.
+For example, boiler plate text for a preview doc might read:
+
+<Callout variant="caution" title="Preview">
+  This feature was released on January 1, 202X and is in preview. To get access to it, reach out to your account manager.
+</Callout>
+
+## Additional guidance for internal contributors [#internal-contributors]
+
+Often, when Internal Contributors want to make a change to a preview doc, they’ll make a fork of the beta-docs-site repo. Because we don’t accept PRs from forked repos, you’ll need to ask the #doc-eng-help Slack channel for access to the repo. Once you receive access, you can make a branch off the beta-docs-site repo. If you've already created a PR from a forked repo, you'll need to close it and create a new PR that branches off the beta-docs-site repo.
```

Comment:
```suggestion
As an internal contributor looking to edit a preview doc, you'll need access to the beta-docs-site repo. If you don't have access, ask the #doc-eng-help Slack channel to add you as a contributor. Once you have access, create your own branch of the `main` branch and make your edits there.
```

@paperclypse What do you think about addressing internal contributors directly? Are we expecting to share this doc with them?

Reply:
@ally-sassman I think it's fine to address individual contributors